src/AssemblerOptions.cpp

```diff
@@ -753,7 +761,30 @@ void AssemblerOptions::MarkerGraphOptions::parseSimplifyMaxLength()
 
 }
 
+void AssemblerOptions::ReadsOptions::parseDesiredCoverageString() {
+    desiredCoverage = std::stoull(desiredCoverageString);
 
+    size_t pos = desiredCoverageString.find_last_of("0123456789");
```

Comment:
You could also use a second argument to `stoull` to get the position of the first non-numeric character. That would also be more robust because the code as written would not deal well with a string like "18G2". It would just interpret it as "18".

Reply:
Nice! I like this.